widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -1307,7 +1279,7 @@ For IE 9 and IE10, need to add padding to the bottom of some dialogs generated w
 }
 
 .pentaho-dialog .button-panel .pentaho-button {
-  margin-left: 10px;
+  margin-left: 10px !important;
```

Comment:
This is not needed, AFAICT. Also, we should not be changing the non-responsive base dialog styles.

Reply:
Reverted.